### PR TITLE
Minor corrections to gcp workload identity docs

### PR DESCRIPTION
**Pull request by akhil-rane**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -211,7 +211,7 @@ Read more about supported clouds by clicking on the links below:
 ## Support Matrix
 Cloud | Mint | Mint + Remove Admin Cred | Passthrough | Manual | Token
 --- | --- | --- | --- | --- | ---
-AWS | Y | 4.4+ | Y | 4.3+ | 4.6+ (expected)
+AWS | Y | 4.4+ | Y | 4.3+ | 4.6+
```
Comment (joelddiaz) [inline README.md:214]: Should be 4.8 when STS token support went GA for AWS.

Comment (akhil-rane) [reply README.md]: My bad, fixed!

**Review by joelddiaz — COMMENTED** [review]
/lgtm

**Comment by akhil-rane** [comment]
/label qe-approved

just some doc fixes, no-qe work

**Comment by akhil-rane** [comment]
/label px-approved

The parent [epic](https://issues.redhat.com/browse/CCO-114) is px-approved by @sferich888

**Pull request merged**